e2e-tests/tls-issue-cert-manager/conf/some-name-tls-issue.yml

```diff
@@ -28,22 +28,17 @@ spec:
             storage: 2Gi
     affinity:
       antiAffinityTopologyKey: "kubernetes.io/hostname"
-  proxysql:
+  haproxy:
```

Comment:
I do not think that we need to remove proxysql we need to add additional test case

Reply:
@hors ProxySQL returned in 8534571ee6cf0ed23f490c8dfec40265f00ee02c